_layouts/home.html

```diff
@@ -31,33 +33,28 @@ <h3>
               {{ post.title | escape }}
             </a>
           </h3>
-          {%- if site.show_excerpts -%}
-            {{ post.excerpt }}
-          {%- endif -%}
+          <div class="post-snippet">
+            {{ post.content | strip_html | truncatewords: 22}}
+          </div>
         </li>
         {%- endfor -%}
       </ul>
   
       {% if site.paginate %}
         <div class="pager">
           <ul class="pagination">
-          {%- if paginator.previous_page %}
```

Comment:
You've removed pagination?

Reply:
Pagination is still enabled but is now handled by the JS in `gsce-custom.js`. The pagination variables that Jekyll provide are quite simplistic https://jekyllrb.com/docs/pagination/#liquid-attributes-available. In order to have consistent styling with Google pagination, I needed to use the pagination variables provided by Jekyll along with some JS manipulation - see `SetupPagination()` in `gsce-custom.js`.